.gitignore

```diff
@@ -11,3 +11,7 @@ obj/
 /Build/*.debug.cfg
 
 *.user
+/Build/TestSound
```

Comment:
It's unclear what is this directory. Do we need it?

Reply:
No it is,  temporary directory